snippets/standard/datetime/json/reading-with-utf8jsonreader-error.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Text;
+using System.Text.Json;
+
+public class Example
+{
+    public static void Main(string[] args)
+    {
+        byte[] utf8Data = Encoding.UTF8.GetBytes(@"""2019/07/26 00:00:00""");
+
+        var json = new Utf8JsonReader(utf8Data);
+        while (json.Read())
+        {
+            if (json.TokenType == JsonTokenType.String)
+            {
+                Console.WriteLine(json.TryGetDateTime(out DateTime datetime));
+                Console.WriteLine(datetime);
+
+                var _ = json.GetDateTime();
+            }
+        }
+    }
+}
+
+// The example displays the following output:
+// False
+// 1/1/0001 12:00:00 AM
+// Unhandled exception. System.FormatException: The JSON value is not in a supported DateTime format.
+//     at System.Text.Json.Utf8JsonReader.GetDateTime()
+//     at Example.Main(String[] args) in D:\console_apps\DateTimeConverterExamples\Program.cs:line 19
```

Comment:
Should we avoid callstack info like this, which is specific to a local repro (like the path info)? Is stating just the exception and the immediate method that caused it sufficient?

Reply:
We've generally displayed all of the information from Exception.Message. But a shorter version is fine, since this is really a bit excessive.

Reply:
> We've generally displayed all of the information from Exception.Message. 

Question: And do we generally include the filepath info in such cases, like D:\console_apps\ etc which are just specific to @layomia's local run? I would have expected that to cause some confusion.